### PR TITLE
Update Shortcut key reference

### LINKED ISSUE #2895: Missing some cutting edge shortcuts
Not sure if there's more shortcuts.

Pause and x3/x4 shortcuts: https://osu.ppy.sh/community/forums/topics/843652

Wiki page: https://osu.ppy.sh/help/wiki/Shortcut_Key_Reference

Reference:

### PR DESCRIPTION
**Pull request by MegaApplePi**
fixes #2895

I think this is all of them. Also updated some others as well.

### CODE REVIEW

File: wiki/Shortcut_key_reference/en.md [hunk 1]
```diff
@@ -131,6 +131,18 @@ These shortcuts work anywhere:
 | `F3` or `F4` | Adjust speed for osu!mania (can be used near the beginning of gameplay). |
 | `Ctrl` + `R` or `` ` `` | Must be held briefly. Quick retry the current map (skips the intro upon retry) |
 
+## Replay mode
+
+*Notice: You must use the cuttingedge release stream to use these.*
```
Comment (cl8n) [inline wiki/Shortcut_key_reference/en.md]: if this type of thing is supposed to be a `Notice` I think I've been mistakenly calling them `Note`s all this time 😨

File: wiki/Shortcut_key_reference/en.md [hunk 2]
```diff
@@ -194,32 +194,32 @@ These shortcuts work anywhere within the beatmap editor:
...
 #### General
 
-| Shortcut | Description |
-| --- | --- |
+| Shortcut | Action |
+| :-- | :-- |
 | `Alt` + `F4` or `Esc` | Return to song selection. |
-| `Ctrl` + `Z` | Undo |
-| `Ctrl` + `Y` | Redo |
+| `Ctrl` + `Z` | Undo. |
```
Comment (MegaApplePi) [inline wiki/Shortcut_key_reference/en.md]: I left the full stops out since they weren't really full sentences imo.

Comment (cl8n) [reply wiki/Shortcut_key_reference/en.md]: well... technically they are, I see what u mean though. I just put them on to be consistent with everything else, I'm fine with either

File: wiki/Shortcut_key_reference/en.md (hunk 1 above)
Comment (peppy) [reply wiki/Shortcut_key_reference/en.md]: i believe "note" is more correct.

"note that"

File: wiki/Shortcut_key_reference/en.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+| Shortcut | Action |
+| :-- | :-- |
+| `+` or `-` | Adjust the local offset (hold `Alt` to change the precision). |
+| `Esc` | Pause gameplay, or return to song selection if watching a replay or Auto or Cinema mod is used. |
+| `Space` | Skip to gameplay. |
+| `Tab` | Toggle the scoreboard. |
+| `Shift` + `Tab` | Toggle hiding of the in-game interface. |
+| `H` | Toggle hiding of the scoreboard and replay interface (when watching a replay or spectating). |
+| `F3` or `F4` | Adjust speed for osu!mania (can be used near the beginning of gameplay). |
+| `Ctrl` + `R` or `` ` `` | Must be held briefly. Quick retry the current map (skips the intro upon retry). |
```
Comment (MegaApplePi) [inline wiki/Shortcut_key_reference/en.md]: It looks like this line is broken on the web. Will open an issue later on osu-web if I remember

File: wiki/Shortcut_key_reference/en.md (hunk 1 above)
Comment (Joehuu) [reply wiki/Shortcut_key_reference/en.md]: https://osu.ppy.sh/help/wiki/Article_Styling_Criteria#notices

We use other words to express how important it is and give attention to.

Comment (Coppertine) [reply wiki/Shortcut_key_reference/en.md]: Isn't this the exact same as NB:

**Pull request merged**